app/views/shared/_sign.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class='sign'>
+  <div class='rh'>
+    <%if user_signed_in?%>
+      <%= button_to "sign out", destroy_user_session_path, method: :delete%>
+    <%else%>
+      <%= link_to  "sign in", new_user_session %>
+    <%end%>
+  </div>
+
+  <div class=total>
+    <p><%=total%></p>
+  </div>
+
+  <div class='lh'>
+    <%= link_to "<", model %>
```

Comment:
- [ ] As required for the project to stick as much as possible to the **[Behance design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)**. kindly ensure that you improve the home page to match the one from the template.

| Template Design | Your Design |
| -------------------- | -------------------- |
| ![Screenshot from 2023-05-31 14-22-19](https://github.com/Tobby8629/Budget_app/assets/83042742/e1a4b6fd-6703-4b3a-9fb5-2b4bc5e3fabc) | ![Screenshot from 2023-05-31 14-22-19](https://github.com/Tobby8629/Budget_app/assets/83042742/10185717-0025-4d81-b534-f7a4888ec193) |